hal/src/nRF52840/image_bootloader.sh

```diff
@@ -17,7 +17,7 @@ function update_bl()
    pushd ../../../bootloader
    make -s PLATFORM_ID=$1 COMPILE_LTO=n $clean all || die
    popd
-   miniz_compress c $bl_dir/platform-$1/bootloader.bin bootloader_platform_$1.bin.miniz || die
+   miniz_compress c $bl_dir/platform-$1-m/bootloader.bin bootloader_platform_$1.bin.miniz || die
```

Comment:
are these scripts part of the CI build? if not, it's a good idea to add them.

Reply:
We are no longer embedding the bootloaders though. I simply fixed them with the new target name in case we do come back to doing so.